cms/tests/test_views.py

```diff
@@ -152,6 +152,19 @@ def test_redirect_with_toolbar(self):
             response = self.client.get(self.get_toolbar_disable_url(page_url))
             self.assertEqual(response.status_code, 302)
 
+    @override_settings(REDIRECT_PRESERVE_QUERY_PARAMS=True)
```

Comment:
Is there also a test that ensures that params are removed on redirect?

Reply:
Nop, it weren't. 

I've added 1 new test. So this PR has 2 new tests:
- test_redirect_not_preserving_query_parameters
- test_redirect_preserving_query_parameters